src/main/resources/l10n/JabRef_da.properties

```diff
@@ -1781,3 +1780,8 @@ should_have_the_first_letter_capitalized=
 Remember_password?=
 
 shared=
+Finished_downloading_full_text_document_for_entry_%0.=
+
+No_entry_with_id_'%0'_for_fetcher_'%1'_was_found.=
+
+Error_while_fetching=Fejl_under_hentning
```

Comment:
Where do you know Dansk from? :innocent: - Are we really sure that there is no equivalent translation somewhere? What do the web-based fetchers output if something goes wrong?